routes/tasks.js

```diff
@@ -115,7 +115,7 @@ router.post('/',  validateTask, asyncHandler(async (req, res, next) => {
     }
 }))
 
-router.put('/:id(\\d+)', csrfProtection, validateEditTask, asyncHandler( async (req,res, next) => {
+router.post('/:id(\\d+)', csrfProtection, validateEditTask, asyncHandler( async (req,res, next) => {
```

Comment:
Don't forget to change to put once we figure out how it will work on the front end